### PR TITLE
Fix/layout component doc strings

### PR DESCRIPTION
**Pull request by Moro-Code**
# Description

No task name just a couple of fixes for the layout component

1. Removed unnecessary div elements so html is more semantic 

Screen Shot 2021-04-07 at 1 09 49 PM

2. Took out start testing button and become testing button out of the layout component. This will not be part of every page

3. There were some spaces at the end of some css classes, took those out

4. modified tests and added one more for a11y when banner is rendered
 

## Acceptance Criteria

N/A

## Test Instructions

1. Clone and run locally see if there is any differences between main.... there shouldn't be 

## Help Requested
N/A

## Checklist

- [x] Strings use placeholders for translation (No hard coded strings)
- [x] Unit tests have been added/updated

## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
 import "../styles/fonts.css";
+import Head from "next/head";
 
 function MyApp({ Component, pageProps }) {
-  return <Component {...pageProps} />;
+  return (
+    <>
+      <Head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      </Head>
```
Comment (pcraig3) [inline pages/_app.js:11]: Good idea to move this here 👍

File: pages/index.js [hunk 2]
```diff
@@ -1,59 +1,73 @@
...
+          <ActionButton
+            text="Start testing"
+            secondary
+            className={"mr-4"}
+            dataCyButton={"become-tester-button"}
+          />
+          <ActionButton
+            text="Become tester"
+            dataCyButton={"start-testing-button"}
+          />
```
Comment (pcraig3) [inline pages/index.js:28]: This is probably "Become a tester", although in [the newer mockups](https://www.figma.com/file/t5uDkPa86nsS3ssFESX2Em/BDM-DECD-Alpha-Site-Sprint-2?node-id=105%3A3801), the buttons are gone anyway. 💣

**Review by pcraig3 — COMMENTED** [review]
Looks like all good changes. @EphraimAssi, can you look through this as well?

**Comment by EphraimAssi** [comment]
It looks good to me too, the code is way more clean.

**Review by EphraimAssi — APPROVED** [review]
Looks good and way more clean.

**Pull request merged**